docs/OnnxReleases.md

```diff
@@ -2,25 +2,43 @@ The ONNX project, going forward, will plan to release roughly on a two month cad
     * And draft a new release statement - https://github.com/onnx/onnx/releases listing out the new features and bug fixes, and potential changes being introduced in the release.
-* Checkout the release tag in a clean branch on your local repo. Make sure all tests pass on that branch.
-* Make sure that the Release version number information is up-to-date in the following places:
+* Create a release branch (rel-1.X.X) from master. Checkout the release tag in a clean branch on your local repo. Make sure all tests pass on that branch.
+* In the release branch, make sure that the Release version number information is up-to-date in the following places:
 [VERSION_NUMBER file](https://github.com/onnx/onnx/blob/master/VERSION_NUMBER) and
 [version.h](../onnx/common/version.h)
 * Make sure that the IR version number and opset version numbers are up-to-date in
 [ONNX proto files](../onnx/onnx.in.proto),
-[Versioning.md](Versioning.md) and
-[helper.py](../onnx/helper.py).
+[Versioning.md](Versioning.md), 
+[schema.h](../onnx/defs/schema.h), 
+[helper.py](../onnx/helper.py) and [helper_test.py](../onnx/helper_test.py).
 * Make sure all operators version are even number, if opset version is odd number then increment it and update all corresponding OPs.
+* Start early to create the release summary (draft).
+
+## Test with TestPyPI
+**Wheel Files**
+* Use `VERSION_NUMBER` like `1.x.0rc1` as release candidate for verification before finally using targeted `VERSION_NUMBER`.
+* Windows
+  * Use GitHub Action (`.github/workflows/release_win.yml`) under onnx repo to produce wheel files for Windows.
+  * After success, upload the produced wheel files manually to TestPyPI: `twine upload --verbose *.whl --repository-url https://test.pypi.org/legacy/ -u PYPI_USERNAME -p PYPI_PASSWORD`.
+* Linux and Mac
+  * Use Travis CI from [onnx/wheel-builder](https://github.com/onnx/wheel-builder) repo to produce and automatiallcy upload wheel files for Linux and Mac.
+  * Update `BUILD_COMMIT` as the commit ID of latest release branch in `.travis.yml` and update `ONNX_NAMESPACE` as `ONNX_REL_1_X` in `config.sh `.
+  * Update `$PYPI_USERNAME` and `$PYPI_PASSWORD` as `- secure: ***` in `.travis.yml`. Create the encrypted variables for these variables by `travis encrypt` in your local machine.
+  Reference: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml
+  * Only `PyPI-test` branch will automatiallcy upload created wheel files to TestPyPI.
+  * Currently Python 3.5 on Mac cannot upload wheel successfully in Travis CI. In that case, you need to upload the created wheel files to AWS S3 bucket, get the wheel from AWS and upload it manually (same as Windows). To upload to AWS, updade your `ARTIFACTS_KEY`, `ARTIFACTS_SECRET` and `ARTIFACTS_BUCKET` by `travis encrypt`. Reference: https://docs.travis-ci.com/user/uploading-artifacts/
+
+**Source Distribution**
```

Comment:
Suggestion: This is for line 50

Since everywhere else for manual twine upload we are already using just the command line for adding url as well as username password we can change the twine upload command for sdist upload as well so that you can remove the step to update the .pypirc file

Reply:
Good idea. I also considered about doing this... Just removed and updated. It should look clearer now. Thank you!